--- META-INF/ruby.gems/gems/sass-3.2.3/lib/sass/importers/filesystem.rb	2012-12-07 12:05:59.000000000 +0000
+++ META-INF/ruby.gems/gems/sass-3.2.3/lib/sass/importers/filesystem.rb	2012-12-07 13:19:42.000000000 +0000
@@ -111,43 +111,12 @@
       def find_real_file(dir, name, options)
         found = possible_files(remove_root(name)).map do |f, s|
           path = (dir == "." || Pathname.new(f).absolute?) ? f : "#{dir}/#{f}"
-          Dir[path].map do |full_path|
-            full_path.gsub!(REDUNDANT_DIRECTORY, File::SEPARATOR)
-            [full_path, s]
+          if full_path = Dir[path].first
+            full_path.gsub!(REDUNDANT_DIRECTORY,File::SEPARATOR)
+	    return full_path, s
           end
-        end
-        found = Sass::Util.flatten(found, 1)
-        return if found.empty?
-  
-        if found.size > 1 && !@same_name_warnings.include?(found.first.first)
-          found.each {|(f, _)| @same_name_warnings << f}
-          relative_to = Pathname.new(dir)
-          if options[:_line]
-            # If _line exists, we're here due to an actual import in an
-            # import_node and we want to print a warning for a user writing an
-            # ambiguous import.
-            candidates = found.map {|(f, _)| "    " + Pathname.new(f).relative_path_from(relative_to).to_s}.join("\n")
-            Sass::Util.sass_warn <<WARNING
-WARNING: On line #{options[:_line]}#{" of #{options[:filename]}" if options[:filename]}:
-  It's not clear which file to import for '@import "#{name}"'.
-  Candidates:
-#{candidates}
-  For now I'll choose #{File.basename found.first.first}.
-  This will be an error in future versions of Sass.
-WARNING
-          else
-            # Otherwise, we're here via StalenessChecker, and we want to print a
-            # warning for a user running `sass --watch` with two ambiguous files.
-            candidates = found.map {|(f, _)| "    " + File.basename(f)}.join("\n")
-            Sass::Util.sass_warn <<WARNING
-WARNING: In #{File.dirname(name)}:
-  There are multiple files that match the name "#{File.basename(name)}":
-#{candidates}
-  This will be an error in future versions of Sass.
-WARNING
-          end
-        end
-        found.first
+	end
+	nil
       end
 
       # Splits a filename into three parts, a directory part, a basename, and an extension
